extensions/warp-ipfs/src/store/conversation.rs

```diff
@@ -341,19 +345,20 @@ impl ConversationDocument {
                                 .iter()
                                 .flat_map(|rec| rec.to_string().as_bytes().to_vec()),
                         )),
-                        (!settings.members_can_add_participants()).then_some(Vec::from_iter(
-                            self.recipients
-                                .iter()
-                                .flat_map(|rec| rec.to_string().as_bytes().to_vec()),
-                        )),
+                        (!settings.user_has_permission(&did, Permission::AddParticipantsToGroup))
+                            .then_some(Vec::from_iter(
+                                self.recipients
+                                    .iter()
+                                    .flat_map(|rec| rec.to_string().as_bytes().to_vec()),
+                            )),
```

Comment:
I do wonder if we should have this here. I guess for normal group conversations this should be alright but for communities this would be redundant. I guess i would have to think more on that when the type actually comes.

Reply:
noted